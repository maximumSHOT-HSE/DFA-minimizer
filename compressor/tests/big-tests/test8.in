11 55 5
1 7 0
1 7 1
1 11 2
1 8 3
1 1 4
2 7 0
2 11 1
2 7 2
2 7 3
2 3 4
3 9 0
3 8 1
3 11 2
3 5 3
3 3 4
4 8 0
4 6 1
4 4 2
4 2 3
4 4 4
5 8 0
5 3 1
5 3 2
5 6 3
5 6 4
6 8 0
6 9 1
6 5 2
6 9 3
6 10 4
7 7 0
7 1 1
7 5 2
7 8 3
7 2 4
8 10 0
8 7 1
8 7 2
8 10 3
8 1 4
9 10 0
9 4 1
9 6 2
9 8 3
9 7 4
10 4 0
10 11 1
10 1 2
10 11 3
10 11 4
11 7 0
11 2 1
11 6 2
11 7 3
11 10 4
9
1
3 
