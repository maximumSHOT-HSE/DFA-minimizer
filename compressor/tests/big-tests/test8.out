11 55 5
7 1 1
10 1 2
1 1 4
8 1 4
11 2 1
4 2 3
7 2 4
5 3 1
5 3 2
2 3 4
3 3 4
10 4 0
9 4 1
4 4 2
4 4 4
6 5 2
7 5 2
3 5 3
4 6 1
9 6 2
11 6 2
5 6 3
5 6 4
1 7 0
2 7 0
7 7 0
11 7 0
1 7 1
8 7 1
2 7 2
8 7 2
2 7 3
11 7 3
9 7 4
4 8 0
5 8 0
6 8 0
3 8 1
1 8 3
7 8 3
9 8 3
3 9 0
6 9 1
6 9 3
8 10 0
9 10 0
8 10 3
6 10 4
11 10 4
2 11 1
10 11 1
1 11 2
3 11 2
10 11 3
10 11 4
9
1
3 
